cosign/README.md

```diff
@@ -0,0 +1,125 @@
+# Cosign Verifier
+
+## Introduction
+
+In order to provide signatures over arbitrary artifacts, Sigstore performs three major operations[1]:
+
+1. OIDC Issuance, vouch that a client is in control of an identity
+2. Associate short-lived public key certificates with these identities (from a Certificate Authority), and publish these certificates to an Identity Log.
+3. Publish a long-lived signature over an artifact (or artifact meta-data) to an Artifact Log, allowing verifiers to check its validity.
+
+The first and second operations are performed by a system called Fulcio, which functions as a Certificate Authority and transparency log for a namespace of OIDC identities (Identity Log).
+The third operation is performed by a system called Rekor, a transparency log for artifact signatures (Artifact Log).
+And cosign is the reference Sigstore client implementation to sign arbitrary artifacts. With these components, Sigstore completed trust setup, signing and verification[2].
+
+```mermaid
+flowchart TB
+  %% OIDC Flow
+  Signer["💻 Signer"] -->|① OIDC Challenge Start| OIDC["@ OIDC Provider"]
+  OIDC -->|② OIDC Response| Signer
+  
+  %% Certificate Issuance via Fulcio
+  Signer -->|③ Issuance Request| Fulcio["📜 Fulcio (Certificate Authority)"]
```

Comment:
What is `Issuance Request`? Do you mean `Certificate Request`? or simply `Request certificate`?

Reply:
Referring [official doc description](https://docs.sigstore.dev/certificate_authority/certificate-issuing-overview/), updated as `certificate request`